src/main/java/org/opensearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -264,10 +264,7 @@ public XContentBuilder errorsAsXContent(RestChannel channel) {
                 case WEAK_PASSWORD:
                 case SIMILAR_PASSWORD:
                     builder.field("status", "error");
-                    builder.field(
-                        "reason",
-                        opensearchSettings.get(ConfigConstants.SECURITY_RESTAPI_PASSWORD_VALIDATION_ERROR_MESSAGE, errorType.message)
-                    );
+                    builder.field("reason", errorType.message);
```

Comment:
Hey @derek-ho, this change looks good to me. Would it be possible to write an integ test that uses the reset password API and show this change working?

You can look at [DashboardsInfoWithSettingsTest](https://github.com/opensearch-project/security/blob/main/src/integrationTest/java/org/opensearch/security/api/DashboardsInfoWithSettingsTest.java) for an example of setting up a node with different settings and using a REST client to hit security APIs.

This logic should be hit during reset_password, so it may be useful to have integration tests that cover different password scenarios. See https://opensearch.org/docs/latest/security/access-control/api/#change-password